event_test.go

```diff
@@ -13,22 +13,10 @@ type EventSuite struct {
 	suite.Suite
 }
 
-type SampleAggregate struct {
```

Comment:
This was used in more test files so it make sense to move this to a common place. Everything that was reused in more than one test were moved to `support_test.go`